src/toolbar.rst

```diff
@@ -29,6 +49,11 @@ Fill More BFS and Less
 You can fill More BFS and Less topology table using the button *Fill More BFS
 and Less* on Toolbar LADM_COL which makes automatic and faster this job.
 
+This are the tables that relate with this function:
+
+masccl --> puntolindero and lindero
+menos --> puntolindero and lindero (holes or intern rings)
```

Comment:
Terreno and Lindero

Reply:
fixed https://github.com/AgenciaImplementacion/Asistente-LADM_COL-docs/pull/9/commits/d670a21d96504dd2dbdc3ce41c16f7b3bcb76255